src/react/workflow/details/Configuration.jsx

```diff
@@ -85,6 +85,7 @@ function Configuration({
               label="Delete Workflow"
               variant="danger"
               onClick={handleOpenDeleteModal}
+              type="button"
```

Comment:
should we add this type in the core-ui then?

Reply:
it's from the HTML specification. Default type is submit. Not sure it's needed in core-ui since it's classic HTML